src/Cli/Command/RevokeCommand.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+/*
+ * This file is part of the Acme PHP project.
+ *
+ * (c) Titouan Galopin <galopintitouan@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace AcmePhp\Cli\Command;
+
+use AcmePhp\Cli\ActionHandler\ActionHandler;
+use AcmePhp\Cli\Repository\RepositoryInterface;
+use AcmePhp\Core\AcmeClientV2Interface;
+use AcmePhp\Core\Exception\Protocol\CertificateRevocationException;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class RevokeCommand extends AbstractCommand
+{
+    /**
+     * @var RepositoryInterface
+     */
+    private $repository;
```

Comment:
You don't really need properties, do you? I would be in favor of using local variables in the execute method.